<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QDialog" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="SettingsTab" name="settingsTab">
        <attribute name="title">
         <string>Settings</string>
        </attribute>
       </widget>
       <widget class="LogTab" name="logsTab">
        <attribute name="title">
         <string>Logs</string>
        </attribute>
       </widget>
       <widget class="AboutTab" name="aboutTab">
        <attribute name="title">
         <string>About</string>
        </attribute>
       </widget>
       <widget class="AdvancedTab" name="advancedTab">
        <attribute name="title">
         <string>Advanced</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SettingsTab</class>
   <extends>QWidget</extends>
   <header>src/window.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogTab</class>
   <extends>QWidget</extends>
   <header>src/window.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AboutTab</class>
   <extends>QWidget</extends>
   <header>src/window.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AdvancedTab</class>
   <extends>QWidget</extends>
   <header>src/window.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
